design-spec/priviledge-decentralized-swupdates.tex

```diff
@@ -193,8 +245,92 @@ \section{Satisfying the requirements}
 
 % How does the different aspects of the design satisfies the requirements.
 
-% It seems that here we'd want to add the same subsections as in the
-% requirements section.
+\subsection{Open}
+\label{sec:sat-open}
+
+This design does not address the definition of minimal requirements for proposal
+submissions that will prevent spam attacks.
+
+This design addresses the requirement that any participant that can submit a
+common transaction should be able to vote on update proposals, by including
+defining a new \emph{vote} transaction type. Furthermore, the fee that these
+vote-transactions pay are calculated in the same way as monetary transactions.
+In this way, stakeholders can vote directly by submitting a vote, or indirectly
+by delegating their voting rights to experts.
+
+\subsection{Democratic}
+\label{sec:sat-decentr-decis-making}
+
+This design addresses the requirements that the community can vote on updates,
```

Comment:
I think it worth's to emphasize that our proposal removes all central authorities (e.g., code maintainer/owner) from the core decision points in the typical software update lifecycle and replaces them with a on-chain voting process.